addon/components/paper-progress-linear.js

```diff
@@ -41,7 +42,7 @@ export default Ember.Component.extend(ColorMixin, {
     }
   }),
 
-  queryMode: Ember.computed('mode', function() {
+  queryMode: computed('mode', function() {
```

Comment:
Please change `queryMode` to `queryModeClass` as it makes this more evident that its purpose is to be used in the template.